app/SearchApi/SearchApi.Web/People/ExecuteSearch.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SearchApi.Web.Controllers
+{
+    public interface ExecuteSearch
```

Comment:
shall we use IExecuteSearch to make it clear that it is an interface?

Reply:
Hehe, I kinda agree with you, but in MassTransit the contract don't start with the I reference....

Reply:
https://masstransit-project.com/MassTransit/usage/producing-messages.html#sending-via-interfaces

Reply:
good to know.